e3db/tests/test_search_integration.py

```diff
@@ -187,20 +187,30 @@ def test_v2_invalid_range(self):
         assert(len(results)==0)
 
     def test_v2_range_end_none(self):
-        q = e3db.types.Search(include_data=True).match(record_types=[self.record_type]).range(zone="PST", start=datetime.now())
```

Comment:
are you going to have to update this in the fall? Can we switch to UTC timezone's or get the current timezone dynamically?

Reply:
definitely just found out I can do use astimezone() to https://docs.python.org/3/library/datetime.html#datetime.datetime.astimezone
I'm gonna make some modifications to the range params so it defaults to this